### DISCUSSION TITLE
Use mini.pick for code actions

### DISCUSSION THREAD

**Discussion by diego-velez** [nvim-mini/mini.nvim#1437] [Q&A] [answered]
Basically the title, is there a way to use `mini.pick` for code actions? I'm already using it for the other LSP actions with `mini.extra`

**Comment by echasnovski** [comment, marked as answer]
As far as I can tell, `vim.lsp.buf.code_action()` works pretty well with `vim.ui.select = MiniPick.ui_select`. So I don't see a reason for a separate picker.
